### PR TITLE
Feature/actual camera

### PR DESCRIPTION
**Pull request by RamonGilabert**

### CODE REVIEW

File: Source/BottomView/BottomContainerView.swift [hunk 1]
```diff
@@ -51,7 +52,17 @@ class BottomContainerView: UIView {
     return configuration
     }()
 
+  lazy var topSeparator: UIView = { [unowned self] in
+    let view = UIView()
+    view.setTranslatesAutoresizingMaskIntoConstraints(false)
+    view.backgroundColor = self.configuration.backgroundColor
+    self.addSubview(view)
```
Comment (richardoti) [inline Source/BottomView/BottomContainerView.swift]: Better to add subviews in ViewDidLoad, so the views would not be added ocassionally, when you just want to access them.

File: Source/BottomView/ButtonPicker.swift [hunk 2]
```diff
@@ -53,7 +53,7 @@ class ButtonPicker: UIButton {
   // MARK: - Configuration
 
   func setupButton() {
-    backgroundColor = .whiteColor()
+    backgroundColor = UIColor.whiteColor()
```
Comment (vadymmarkov) [inline Source/BottomView/ButtonPicker.swift]: What's wrong with `.whiteColor()` :smiley:

Comment (RamonGilabert) [inline Source/BottomView/ButtonPicker.swift]: Oh, probably I changed it to something else and then put white color again! :dancer:

**Comment by vadymmarkov** [comment]
@RamonGilabert Overall looks good for me. Nice work man! Is everything ready to merge?

**Comment by RamonGilabert** [comment]
@vadymmarkov I'm changing 3 things right now and it's going to be ready to merge and use! :)

File: Source/BottomView/BottomContainerView.swift [hunk 3]
```diff
@@ -143,30 +165,54 @@ class BottomContainerView: UIView {
...
+        animateImageView(imageWrapper.firstImageView)
+      }
     case 0:
       imageWrapper.firstImageView.image = nil
-      imageWrapper.secondImageView.image = nil
     case 2:
-      imageWrapper.firstImageView.image = array[0] as? UIImage
-      imageWrapper.secondImageView.image = array[1] as? UIImage
+      imageWrapper.firstImageView.image = array[1] as? UIImage
+      imageWrapper.secondImageView.image = array[0] as? UIImage
```
Comment (richardoti) [inline Source/BottomView/BottomContainerView.swift]: array.first can be used here

**Pull request merged**